packages/rollup-full-node/src/app/web3-rpc-handler.ts

```diff
@@ -721,9 +721,10 @@ export class DefaultWeb3Handler
     // Generate the calldata which we'll use to call our internal execution manager
     // First pull out the `to` field (we just need to check if it's null & if so set ovmTo to the zero address as that's how we deploy contracts)
     const ovmTo = ovmTx.to === null ? ZERO_ADDRESS : ovmTx.to
+    const ovmFrom = ovmTx.from === undefined ? ZERO_ADDRESS : ovmTx.from
```

Comment:
Hmm-it seems a little weird to be setting the OVM's from transaction to the zero address--this would never have a valid signature, right?

Reply:
Yep! But transactions sent to `sendTransaction` don't have signatures attached to them yet.

The node is suppose to keep track of which keys are "unlocked" and if a transaction's `from` address is unlocked the node will sign the transaction. In our implementation we're just treating all transactions as valid when running in test mode and ignoring lock/unlock logic...

Reply:
Ahhhh, got it!

Reply:
Ahhh got it!